src/ResourceManager/Network/Commands.Network/VirtualNetwork/NewAzureVirtualNetworkCommand.cs

```diff
@@ -76,12 +76,17 @@ public class NewAzureVirtualNetworkCommand : VirtualNetworkBaseCmdlet
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "A hashtable which represents resource tags.")]
         public Hashtable Tag { get; set; }
-        
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A switch parameter which represents whether DDoS protection is enabled or not. It can only be turned on if a DDoS Protection Plan is associated with the virtual network.")]
```

Comment:
Is this a breaking change?  If so, you need to ensure that you document this in the 'upcoming braking changes' document in network so that customers know the change is breaking.

-Also, since the protection plan is required, You should create a new parameter set that makes both of these parameters mandatory

Reply:
This functionality was only in private preview. We have never GAd this ability. However, we will add it in upcoming braking change. We will also create a new parameter set.

Reply:
We will add the parameter set in the next release. We added the breaking change documentation